src/Apps/Artwork/Components/ArtworkAuctionCreateAlertHeader/ArtworkAuctionCreateAlertHeader.tsx

```diff
@@ -114,10 +114,16 @@ const ArtworkAuctionCreateAlertHeader: FC<ArtworkAuctionCreateAlertHeaderProps>
     >
       <GridColumns py={[4, 4, 6]}>
         <Column span={12}>
-          <Text variant={["md", "md", "lg"]} textAlign={["left", "center"]}>
+          <Text
+            variant={["lg-display", "md", "lg"]}
+            textAlign={["left", "center"]}
+          >
             Bidding for <i>{artwork.title?.trim()}</i>
             {artistName} has ended.
           </Text>
+          <Media lessThan="sm">
+            <Spacer y={1} />
+          </Media>
```

Comment:
I believe here instead of using `<Media />` we can do something like this:
`<Spacer y={1} display={["none", "block"]}  /> `

Reply:
Ah no, <Spacer /> doesn't have `display` property
Then it'd be
```
<Text
    variant={["lg-display", "md", "lg"]}
    textAlign={["left", "center"]}
    my={[1, 0]}
>
```